lib/errors/missing-param.js

```diff
@@ -0,0 +1,11 @@
+class MissingParamError extends Error {
+  constructor(param) {
+    super(`'${param}' is required`);
+  }
+
+  get status() {
+    return 404;
```

Comment:
This should probably be `400`, no?

Reply:
yep - that was a TODO in my head, which I forgot to do